app/javascript/components/components/Header.js

```diff
@@ -1,9 +1,46 @@
 import React, { Component } from 'react'
+import { Nav, NavItem } from 'reactstrap'
 
 export default class Header extends Component {
   render() {
+    const {
+      logged_in,
+      current_user,
+      new_user_route,
+      sign_in_route,
+      sign_out_route
+    } = this.props
+    console.log("logged_in:", logged_in)
+    console.log("current_user:", current_user)
     return (
-      <div>Header</div>
+      <Nav>
+        <NavItem>
+          <a href={"/"} className="nav-link">Home</a>
+        </NavItem>
+        <NavItem>
+          <a href={"/lastcallindex"} className="nav-link">All Cocktails</a>
+        </NavItem>
+        {logged_in &&
+          <NavItem>
+            <a href={'/lastcallprotectedindex'} className="nav-link">My Cocktails</a>
```

Comment:
Utilize the NavLink from react router dom for internal navigation

Reply:
We had to leave some as href because the login and signup pages required a manual refresh otherwise.